lane.py

```diff
@@ -548,32 +548,20 @@ def __init__(self,
         # e.g. "All Science Fiction"
         self.include_all_feed = include_all
 
-        def set_from_parent(field_name, value, default=None):
-            if value is None:
-                if self.parent:
-                    value = getattr(self.parent, field_name, default)
-                else:
-                    value = default
-            setattr(self, field_name, value)
-
-        def set_list(field_name, value, default=None):
-            if value is None:
-                if self.parent:
-                    value = getattr(self.parent, field_name, default)
-                else:
-                    value = default
-            if isinstance(value, basestring):
-                value = [value]
-            set_from_parent(field_name, value, default)
-
         if isinstance(age_range, int):
             age_range = [age_range]
         if age_range is not None:
             age_range = sorted(age_range)
-        set_from_parent('age_range', age_range)
+        self.set_from_parent('age_range', age_range)
 
         self.audiences = self.audience_list_for_age_range(audiences, age_range)
 
+
+        def set_list(field_name, value, default=None):
```

Comment:
I made a change to this in removing the repeat code from `set_from_parent()`, since its called in actual `set_from_parent()` call in the end.